### PR TITLE
Rename abstract_action.h as abstract_action_base.hpp

### PR DESCRIPTION
**Pull request by corot**
(to mimic abstract_execution_base.h)
Also unify headers definitions and namespace indentation

### CODE REVIEW

**Comment by spuetz** [comment]
So its a renameing and a cleanup of indentation / code style. The renaming shouldn't conflict with other released packages, since it is, as far as I know, only used internally.

**Review by spuetz — CHANGES_REQUESTED** [review]

File: mbf_abstract_nav/src/robot_Information.cpp [hunk 1]
```diff
@@ -39,7 +39,8 @@
 #include "mbf_abstract_nav/robot_information.h"
```
[the remark this answers is not in the document]
Comment (spuetz) [reply mbf_abstract_nav/src/robot_Information.cpp:39]: Maybe `ActionBase` is even better than `AbstractActionBase`, what do you think?

[the remark this answers is not in the document]
Comment (corot) [reply mbf_abstract_nav/src/robot_Information.cpp:39]: > Could you also rename robot_(I)nformation.cpp to robot_(i)nformation.cpp. So make the capital letter to a small letter?

oum...  never noticed; done

[the remark this answers is not in the document]
Comment (corot) [reply mbf_abstract_nav/src/robot_Information.cpp:39]: > Maybe `ActionBase` is even better than `AbstractActionBase`, what do you think?

AbstractActionBase matches AbstractExecutionBase... so to me looks mor simetric

**Pull request merged**